sdk/src/main/java/ly/count/android/sdk/ModuleEvents.java

```diff
@@ -156,18 +156,22 @@ public void recordEventInternal(@Nullable final String key, @Nullable Map<String
         _cly.moduleUserProfile.saveInternal();
 
         if (visibilityTracking) {
-            String appInBackground = deviceInfo.isInBackground();
-            int state = 1; // in foreground
-            if ("true".equals(appInBackground)) {
-                state = 0; // in background
-            }
-            L.d("[ModuleEvents] recordEventInternal, Adding visibility tracking to segmentation app in background:[" + appInBackground + "] cly_v:[" + state + "]");
-
             if (segmentation == null) {
                 segmentation = new HashMap<>();
             }
 
-            segmentation.put(VISIBILITY_KEY, state);
+            if (ModuleViews.VIEW_EVENT_KEY.equals(key) && !segmentation.containsKey("visit")) {
```

Comment:
was it visit or dur that exclusive? or both?

Reply:
visit is exclusive for the start views